mssql/base.py

```diff
@@ -95,7 +103,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         'BooleanField': 'bit',
         'CharField': 'nvarchar(%(max_length)s)',
         'DateField': 'date',
-        'DateTimeField': 'datetime2',
+        'DateTimeField': 'datetimeoffset',
```

Comment:
Thank you and sorry for the delayed reply - yes making this optional sounds like a good idea so that those upgrading the backend on an existing project have to explicitly acknowledge the change - it should also be clearly marked in release notes. Is there a ticket to track this?

Reply:
Sorry for the late reply, but yes this is something that should be clearly marked in the release notes and we just added a ticket to track this. We are also planning on adding some documentation to the wiki on `USE_DATETIMEOFFSET` and linking to that page from the README.md.

Reply:
c.f. https://github.com/microsoft/mssql-django/pull/160 which makes it optional (thanks!) and also https://github.com/microsoft/mssql-django/wiki/Timezone-Support

Reply:
I also want to stress that something like `USE_DATETIMEOFFSET` would by very helpful. We have a lot of datetime2 columns scattered around and before migrating them, we also have to check other systems relying on this database and what happens to them if we migrate.

Reply:
I agree with @kummerer94 
A flag `USE_DATETIMEOFFSET` would be extremely important for backward compatibility.
We, too, have a lot of datetime2 fields. We use some technology like partitioning (done via raw migrations), and after upgrading to 1.3 it broke our migrations.

Going back to 1.2 is a problem because we are already leveraging fixes from 1.3 (such as support of jsonfield in Azure SQL Managed Instances).
Manually altering columns to datetimeoffset would also be extremely difficult because we have tables with billions of rows.

Reply:
I think we have to open a separate issue for this because I am not sure if this was seen by the contributors here.